Sources/Authentication/AuthenticationHandler.swift

```diff
@@ -57,12 +63,36 @@ public final class AuthenticationHandler: NSObject {
     ///     contextProvider: contextProvider ?? ASPresentationAnchor()
     /// )
     /// ````
-    /// - Parameter configuration: The only way you can using this lib, look at example
+    /// - Parameter configuration: One way you can using this lib, look at example
     /// - Parameter contextProvider: Provide a window to show the login, look at example
     public init(configuration: Configuration, contextProvider: ASPresentationAnchor) {
```

Comment:
I think we can remove contextProvider from the initialiser if we want to. Im just doing it this way in Ejerskift. Isnt it possible to initialise ASPresentationAnchor() in this lib instead? 
Or maybe I just dont understand why its useful to inject

<img width="666" alt="image" src="https://user-images.githubusercontent.com/32563707/218456390-aef9acb3-ab46-47ed-bfe3-cd31960af44e.png">

Reply:
I will be an issue since I need ASPresentationAnchor for assigning it in the lib and if you do it like that it might be fine but that means window and you are creating a new window and it might get to a multiple window problem specially in iPad.

Maybe you can use something like this to catch the window

> var globalPresentationAnchor: ASPresentationAnchor? = nil
class SceneDelegate: UIResponder, UIWindowSceneDelegate {
    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
        // ...            
        globalPresentationAnchor = window
    }
}